src/app/pages/PerpetualPage/hooks/graphql/useGetTraderEvents.ts

```diff
@@ -57,6 +82,15 @@ export enum Event {
   UPDATE_MARGIN_ACCOUNT = 'UPDATE_MARGIN_ACCOUNT',
 }
 
+const getTotalCountFieldName = (event: Event) => {
+  switch (event) {
+    case Event.TRADE:
+      return 'tradesTotalCount';
+    default:
+      return 'tradesTotalCount';
+  }
+};
```

Comment:
No function needed here you can just add an object map that uses the Event enum as key. There should be no default value as the count is only supported for the trades and positions right now.